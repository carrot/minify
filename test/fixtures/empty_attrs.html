<div id='' class='wow'>noid</div>
