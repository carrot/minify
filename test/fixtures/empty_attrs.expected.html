<div class="wow">noid</div>
